Libraries/Components/FocusableWindows/FocusableWindows.windows.js

```diff
@@ -0,0 +1,331 @@
+          ReactNative.findNodeHandle(this._focusable),
+          UIManager.WindowsControl.Commands.focus,
+          null);
+      }
+    }
+
+    blur() {
+      if (this._focusable) {
+        UIManager.dispatchViewManagerCommand(
+          ReactNative.findNodeHandle(this._focusable),
+          UIManager.WindowsControl.Commands.blur,
+          null);
+      }
+    }
+
+    setNativeProps(nativeProps) {
+      let focusableProps = {};
+      let componentProps = {};
+
+      let atLeastOneFocusableProp = false;
+      let atLeastOneComponentProp = false;
+
+      if (nativeProps) {
+        for (const key in nativeProps) {
+          if (key in FocusableWindowsTemplate.focusablePropTypes) {
+            // Property supported by WindowsControl
+            focusableProps[key] = nativeProps[key];
+            atLeastOneFocusableProp = true;
+          } else if (key !== 'style') {
+            // Property supported by Component 
+            componentProps[key] = nativeProps[key];
+            atLeastOneComponentProp = true;
+          } else {
+            // Style case is special because it has to be split:
+            // - "transform" should go to WindowsControl
+            // - everything else goes to Component
+            // The difference (compared with normal property processing) is that no flattening/style id resolution is needed
+            const styles = nativeProps['style'];
+            if (styles) {
+              let focusableStyle = {};
+              let childStyle = {};
+              for (const styleName in styles) {
+                if (styleName === 'transform') {
+                  focusableStyle[styleName] = styles[styleName];
+                  atLeastOneFocusableProp = true;
+                } else {
+                  childStyle[styleName] = styles[styleName];
+                  atLeastOneComponentProp = true;
+                }
+              }
+              focusableProps['style'] = focusableStyle;
+              componentProps['style'] = childStyle;
+            }
+          }
+        }
+        
+        if (this._focusable && atLeastOneFocusableProp) {
+          this._focusable.setNativeProps(focusableProps);
+        }
+
+        if (this._component && atLeastOneComponentProp && this._component.setNativeProps) {
+          this._component.setNativeProps(componentProps);
+        }
+      }
+    }
+  }
+
+  // Prop types are a combination of what Component and WindowsControl expose
+  FocusableWindows.propTypes = Object.assign({}, Component.propTypes, FocusableWindowsTemplate.focusablePropTypes);
+
+  return FocusableWindows;
+}
+
+// WindowsControl implementation peculiarities make it check for presence of RCTView like properties in propTypes,
+// even though those are not really supported/implemented.
+// We add those properties for the sake of the native component property validation.
+FocusableWindowsTemplate.propTypes = Object.assign({}, ViewPropTypes, FocusableWindowsTemplate.focusablePropTypes);
```

Comment:
Why to put this immediately below the definition of FocusableWindowsTemplate rather than spreading the definition of FocusableWindowsTemplate out across the file?